.editorconfig

```diff
@@ -9,9 +9,12 @@ indent_size = 4
 end_of_line = lf
 insert_final_newline = true
 trim_trailing_whitespace = true
+block_comment_start = /*
+block_comment = *
+block_comment_end = */
 
 [*.php]
-max_line_length = 150
+max_line_length = 223
```

Comment:
Is this a hot fix?

Reply:
Yes. In docblocks the whole function signature must be on 1 line.

Reply:
Let's keep it at 150?
But set a custom limit for that one file to 1000?

Reply:
I think https://github.com/greut/eclint does not have that feature.

Getting it done in a minute ... 🏃🏻‍♂️